src/tests/Common/Coreclr.TestWrapper/CoreclrTestWrapperLib.cs

```diff
@@ -426,8 +429,8 @@ public static bool TryPrintStackTraceFromCrashReport(string crashReportJsonFile,
                 Console.WriteLine($"Error reading {crashReportJsonFile}: {ex.ToString()}");
                 return false;
             }
-            dynamic crashReport = JsonSerializer.Deserialize<JsonObject>(contents);
-            var threads = crashReport["payload"]["threads"];
+            var crashReport = JsonNode.Parse(contents)!;
+            var threads = (JsonArray)crashReport["payload"]!["threads"]!;
```

Comment:
Hopefully we will not see null objects where the `!` was added.

Reply:
Correct. And if we do, we would throw an exception anyway, so I figure any further checking is a waste.